content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,221 @@
+
+The main section, beneath the user's information, houses all the products in a Kanban view. Each
+product card displays the name, cost, vendor, photo, and description of the product. If the product
+is configured as new, it also displays a :guilabel:`New` tag.
+
+.. image:: orders/dashboard.png
+   :align: center
+   :alt: The *Lunch* app dashboard with all areas highlighted.
+
+.. note::
+   Anywhere the vendor is listed in the *Lunch* app, such as the Kanban product cards, both the name
+   of the vendor appears as well as their phone number.
+
+The products can be displayed in a list view by clicking the :guilabel:`≣ (four parallel lines)`
+icon in the top-right.
+
+Placing orders
+==============
+
+To place a lunch order, navigate to the main :guilabel:`Order Your Lunch` dashboard by opening the
+*Lunch* app. This is also accessed by navigating to :menuselection:`Lunch app --> My Lunch: New
+Order`.
+
+Add products to an order
+------------------------
+
+Click on a product to order, and the product appears in a :guilabel:`Configure Your Order` pop-up
+window.
+
+At the top of the pop-up window is the product image, name, and price. Beneath that are any
+available :ref:`extra items or options <lunch/extras>`. Click the checkbox next to any extras to add
+them to the order.
+
+Each extra is organized by a category, and lists the name and price for each extra. As extras are
+selected, the displayed price at the top of the pop-up window updates to reflect all current
+selections.
+
+Beneath the extras is the :guilabel:`Description` of the product, followed by a :guilabel:`Notes`
+field. The :guilabel:`Notes` field is used to enter any information to be sent to the vendor
+regarding the order, such as any special requests or food allergies.
+
+When all selections for the product have been made, click the :guilabel:`Add To Cart` button in the
+lower-left of the pop-up window. To cancel the order, click the :guilabel:`Discard` button.
+
+.. image:: orders/pizza.png
+   :align: center
+   :alt: The pop-up window for a personal pizza, with all the extras highlighted and selected.
+
+Errors
+~~~~~~
+
+Depending on how the various :ref:`extras <lunch/configure-extras>` are configured for a vendor, it
+is possible to receive an error when attempting to add products to the cart.
+
+An error can occur when a product is configured to require a selection be made for an extra, but no
+selection is made for that extra category.
+
+When this occurs, a :guilabel:`Validation Error` pop-up window appears. The error is briefly
+explained in the pop-up window. Click :guilabel:`Close` to close the window, and make any necessary
+changes to the :guilabel:`Configure Your Order` pop-up window.
+
+.. example::
+   The vendor, The Pizza Palace, provides a free beverage with any purchase. Their products are
+   configured so that a beverage selection is required when adding one of their products to the
```

Comment:
```suggestion
   configured so that a beverage selection is **required** in the :guilabel:`Extras` field *before* adding one of their products to the
```